packages/CopyInput/src/CopyInput.styles.js

```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components";
+import tokens from "@paprika/tokens";
+
+export const CopyInput = styled.div`
+  display: flex;
+  [data-pka-anchor="heading"] {
+    margin-bottom: 0;
+  }
```

Comment:
Confused by this...? Was it an experiment?

Reply:
hmm mustve been leftover from a copy pasta